tests/safety/common.py

```diff
@@ -16,18 +15,22 @@ class UNSAFE_MODE:
   DISABLE_STOCK_AEB = 2
   RAISE_LONGITUDINAL_LIMITS_TO_ISO_MAX = 8
 
+def len_to_dlc(length):
```

Comment:
same comments as in `helpers.py`. let's get rid of these (re)definitions and define the helpers for this in `__init__.py`